starter-code/iteration-2/todoInterface.ts

```diff
@@ -3,3 +3,11 @@
 // - Method for adding task where will receive a string, doesn't return anything.
 // - Method for listing all task in the console, doesn't return anything.
 // - Method for delete a task where will receive a string, doesn't return anything.
+
+
+export interface Todo {
+  myArray: string[],
```

Comment:
Remember the Array type takes an extra type as parameter to fix the type of its elements, which is represented using angle brackets.

So your line should actually be like this:

```myArray: Array<string>; ```